services/ml/compilation_delegate_ie.cc

```diff
@@ -235,6 +235,13 @@ int32_t CompilationDelegateIe::CreateBlob(
     DLOG(INFO) << "Create blob with size " << blob->size()
                << " for operand index " << index;
     if (fp32_precision) {
+      const mojom::OperandValueInfoPtr& info =
+        model->values[base::NumberToString(index)];
+      size_t size = product(operand->dimensions);
```

Comment:
We have defined data type with https://github.com/otcshare/chromium-src/blob/webml/services/ml/public/mojom/model.mojom#L14, does it meet your requirement?

Reply:
> We have defined data type with https://github.com/otcshare/chromium-src/blob/webml/services/ml/public/mojom/model.mojom#L14, does it meet your requirement?

Thanks! Updated now. @fujunwei